frontend/src/css/components/_notification.scss

```diff
@@ -0,0 +1,34 @@
+.review-notification{
+    display: flex;    
+    background-color: #E7EDF3;
+    border-radius: 10px;
+    padding: 20px 30px ;
+
+
+    &__profile-wrap{
+        display: flex;
+        justify-content: flex-start;
+    }
+
+    &__review-wrap{
+        display: flex;
+        justify-content: flex-end;
+        margin-left: auto;
+    }
+
+    &__username {
+        margin-left: 10px;
+        padding-top: 5px;
+        color: #2F80ED;
+    }
+
+    &__article{
+        margin-left: 5px;
+        padding-top: 5px;
+    }
+
+    .button__small--hollow{
```

Comment:
I don't think you need to rewrite the button CSS. Remove this.

Reply:
As per the figma design , it has a purple border. So i had to add that

Reply:
Yes. Remove it. I just noticed the buttons are not the same in design and code. I'll fix the Figma buttons later on.